scripts/azure-dev/AzureDev.Azure.psm1

```
@@ -1781,6 +1787,337 @@ function Get-AzureDevVmAdminSshPublicKeys {
+      'list',
+      '--subscription',
+      $Config.SubscriptionId,
+      '--resource-group',
+      $Config.ResourceGroup,
+      '--nsg-name',
+      $names.networkSecurityGroup,
+      '--output',
+      'json'
+    ) -Json
+  } catch {
+    return @()
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-AzureDevSshAccessRules' returns an object of type 'System.Object[]' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)